core/controllers/acl_decorators_test.py

```diff
@@ -29,8 +32,10 @@
 from core.tests import test_utils
 import feconf
 
+# pylint: disable=wrong-import-order
```

Comment:
Done but I couldn't remove all the pragmas because isort and pylint are suggesting different things, i.e if I remove the pragma and follow what pylint suggests, isort gives an error.